README.md

```diff
@@ -2,22 +2,22 @@
 
 ## Installation
 
-ACE is installed as pe-ace-server as part of Puppet Enterprise.
+The Agentless Catalog Executor (ACE) is installed as pe-ace-server as part of Puppet Enterprise (PE).
```

Comment:
@DavidS is this built PE or is it something the user has to install?

Reply:
built-in